### BIG diffrences, this is a diff between CLUS "Constant Variable NIL" entry and topic selection in CLHS http://www.lispworks.com/documentation/lw50/CLHS/Body/a_nil.htm
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:constant_variables:nil.txt	2017-04-27 21:57:12.000000000 +0100
+++ saved-pages-clus/cl:constant_variables:nil.txt	2017-04-27 21:57:12.000000000 +0100
@@ -1,4 +1,68 @@
+Constant Variable NIL
+Constant Value
+nil.
+Description
+nil represents both boolean (and generalized boolean) false and the empty list.
+Examples
+nil 
+→
+NIL
+() 
+→
 NIL
-Please select which reference to NIL you intended
 
-Constant VariableTypeGlossary Entry
\ No newline at end of file
+'nil 
+→
+NIL
+'() 
+→
+NIL
+See Also
+ t
+Notes
+None.
+Editor Notes
+The symbol nil is also sometimes used for other purposes as well. For example, as a designator (e.g. a stream designator).
\ No newline at end of file
